From a5362850b023aa1dadb3d57339258bd5cd829e18 Mon Sep 17 00:00:00 2001
From: Yukang Zhong <yukang.zhong@amlogic.com>
Date: Tue, 8 Dec 2015 19:27:35 +0800
Subject: [PATCH 5922/5965] PD#116238: remote: repair data hardware
 verification

commit a287ac15bfee8a7462e0f3db169984160145fbfc

affect the software decode ,data hardware verification

Note: check should be on register config

Change-Id: Id12b8f7c95a5786077fba01a95c7306f752eb6f6
---
 drivers/amlogic/input/new_remote/remote_main.c | 12 +-----------
 1 file changed, 1 insertion(+), 11 deletions(-)
 mode change 100755 => 100644 drivers/amlogic/input/new_remote/remote_main.c

diff --git a/drivers/amlogic/input/new_remote/remote_main.c b/drivers/amlogic/input/new_remote/remote_main.c
old mode 100755
new mode 100644
index 45ff523b6a1b..cae79d098862
--- a/drivers/amlogic/input/new_remote/remote_main.c
+++ b/drivers/amlogic/input/new_remote/remote_main.c
@@ -307,10 +307,6 @@ static void remote_release_timer_sr(unsigned long data)
 	//key report release use timer interrupt
 	remote_data->key_release_report = remote_report_release_key[remote_data->work_mode];
 	remote_data->key_release_report(remote_data);
-	if (remote_data->hardware_check_enable)
-	{
-		am_remote_write_reg(OPERATION_CTRL_REG1,0x9f50);
-	}
 }
 
 static irqreturn_t remote_interrupt(int irq, void *dev_id)
@@ -544,13 +540,7 @@ static long remote_config_ioctl(struct file *filp, unsigned int cmd, unsigned lo
 
 		case REMOTE_IOC_HARDWARE_CHECK_ENABLE:
 			ret = copy_from_user(&remote->hardware_check_enable, argp, sizeof(long));
-			if (remote->hardware_check_enable)
-			{
-				am_remote_write_reg(OPERATION_CTRL_REG1,0x9f50);
-					/*{OPERATION_CTRL_REG1,0x9f40},// boby long decode (8-13)*/
-			}
-
-
+			break;
 
 	}
 	//output result
-- 
2.19.0

